lib/babel-istanbul-instrumenter.js

```diff
@@ -0,0 +1,349 @@
+/* global Symbol */
```

Comment:
I added a comment as to where the code came from and did some minor cleanup on it.